source-academy/frontend
# Fixed CSE Machine Display on Mobile Devices

### Description

Closes #2696 

Running the CSE machine on mobile used to switch the active tab over to code editor instead of viewing the CSE Machine. Now added a check so user would not toggle to the code editor if the user was on the CSE machine tab. 

### Type of change


### How to test

1. Switch to Source-4 on Mobile
2. Run CSE Machine on Mobile
3. CSE Machine should show up on your screen instead of returning to the code editor

### Checklist
